src/base/action.lua

```diff
@@ -204,6 +204,21 @@
 	end
 
 
+---
+-- Determines if an action is headless or not.
+-- Headless actions prevent various built-in messages from printing.
+---
+
+	function action.isHeadless(self)
+		if not self then
+			self = action.current() or {}
+		end
+		if self.headless == true then
```

Comment:
wouldn't ```return self.headless``` be the same? or if you really wanted to enforce a boolean return: ```return self.headless == true```

Reply:
Correct, I suppose `if self.headless then ...` would also work if one wanted to allow values like `1` to be used to enable headless mode as well; I've only been messing with Lua for a few days, so I'm not extremely familiar with any language specific tricks yet.